sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -1823,3 +1824,32 @@ object OptimizeLimitZero extends Rule[LogicalPlan] {
       empty(ll)
   }
 }
+
+/**
+ * Split [[Expand]] into several splitExpand if the projection size of Expand is larger
```

Comment:
I think you mean "Split [[Expand]] into several [[Expand]] ...". There isn't a class called `splitExpand`.

Reply:
ok